src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -51,8 +54,32 @@ function WorkspaceWorkflowsPage({policy, betas, route}: WorkspaceWorkflowsPagePr
 
     const onPressAutoReportingFrequency = useCallback(() => Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_AUTOREPORTING_FREQUENCY.getRoute(policy?.id ?? '')), [policy?.id]);
 
-    const items: ToggleSettingOptionRowProps[] = useMemo(
-        () => [
+    const authorizedPayerEmail = policy?.reimburserEmail;
+    const displayNameForAuthorizedPayer = useMemo(
+        () => PersonalDetailsUtils.getPersonalDetailByEmail(authorizedPayerEmail ?? '')?.displayName ?? authorizedPayerEmail,
```

Comment:
The OpenPolicyWorkflowsPage is sending the `reimburserAccountID` directly to the `policy_` key so we can get personal details directly without the need for the email search.